README.md

```diff
@@ -0,0 +1,89 @@
+console.redhat.com CloudEvents for Java
```

Comment:
It's probably a good idea to review the rendered result of this file:
https://github.com/RedHatInsights/event-schemas-java/blob/43055ad449dd942206f8ea540599f2aa99e70bfc/README.md